domain/media/src/test/kotlin/com/giraffe/media/collections/usecase/ClearCollectionsCacheUseCaseTest.kt

```diff
@@ -0,0 +1,21 @@
+package com.giraffe.media.collections.usecase
+
+import com.giraffe.media.collections.repository.CollectionsRepository
+import io.mockk.coVerify
+import io.mockk.mockk
+import kotlinx.coroutines.test.runTest
+import org.junit.jupiter.api.Test
+
+class ClearCollectionsCacheUseCaseTest {
+    private val collectionsRepository: CollectionsRepository = mockk(relaxed = true)
+    private val clearCollectionsCacheUseCase = ClearCollectionsCacheUseCase(collectionsRepository)
+
+    @Test
+    fun `should call repository when invoked`() = runTest {
```

Comment:
```suggestion
    fun `should call clear collection cache method in repository when invoke the usecase `() = runTest {
```

I think this name is more express